angular/projects/admin/src/app/admin/document/document-form/document-form.component.html

```diff
@@ -56,6 +56,31 @@
                                       value="{{option.key}}">{{option.value}}</mat-radio-button>
                   </mat-radio-group>
                 </mat-form-field>
+
+                <!-- Date Picker -->
+                <mat-form-field *ngSwitchCase="'date'">
+                  <mat-placeholder>{{field.title}}</mat-placeholder>
+                  <mat-datetimepicker-toggle [for]="datePicker" matSuffix></mat-datetimepicker-toggle>
+                  <mat-datetimepicker #datePicker type="date" openOnFocus="true"></mat-datetimepicker>
+                  <input matInput formControlName="{{field.key}}" [matDatetimepicker]="datePicker" autocomplete="false">
+                </mat-form-field>
+
+
+                <!-- Time Picker -->
+                <mat-form-field *ngSwitchCase="'time'">
+                  <mat-placeholder>{{field.title}}</mat-placeholder>
+                  <mat-datetimepicker-toggle [for]="timePicker" matSuffix></mat-datetimepicker-toggle>
+                  <mat-datetimepicker #timePicker type="time" openOnFocus="true"></mat-datetimepicker>
+                  <input matInput formControlName="{{field.key}}" [matDatetimepicker]="timePicker" autocomplete="false">
+                </mat-form-field>
```

Comment:
We can remove the picker that only selects "time" for now. It is unclear when a time without a date would be.

Reply:
That means that you can remove this line too: https://github.com/oddbit/tanam/blob/0e4996fc5c5828aafed7e2956c084ebad034d0b3/models/document-type.models.ts#L7

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/56/commits/15d400fcc5d319e6335e4dea8f6012c9713a2ed7